### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Benefits Management Tools, VA Letters and Documents

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#102994]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I am very sorry for the slow written feedback here! I wanted to chat with a couple of other accessibility folks to get their perspective on the download link experience you have in your prototype, and then forgot to follow through with my written feedback. Please reach out in Slack if you have any questions, or if my delay caused any issues for you.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Consider:** My biggest point of concern in the meeting was the [Benefit Summary and Service Verification Letter link](https://www.figma.com/proto/61I2UcRPJDXdw0xCl9BlEe/Benefits-Letters?page-id=1463%3A2105&node-id=2154-9229&p=f&viewport=-3266%2C843%2C1&t=JNAA3hgzyUm41myW-1&scaling=min-zoom&content-scaling=fixed&starting-point-node-id=2154%3A8970&show-proto-sidebar=1), because I can't think of a comparable experience where a checkbox group is followed by a link.
   
   I do think a download link is the right element here, but I'd encourage you to explore changing the link text to reflect the choices made in the checkbox group. I'm imagining something like "Benefit Summary and Service Verification Letter with [number] sections included (PDF)" or... something like that.
   
   My thought process here: I'm worried that it might not be clear to users that their selections in the checkbox are reflected in what's on the other side of the download link --- and if there's any kind of lag between checking a box and the download link matching what they want, providing some indication to users when it's ready.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by 1Copenut** [comment]
@briandeconinck We have some ideas we're baking right now. Will ping you next week when we've got them fully thought through.

/cc @sh0wn

**Comment by 1Copenut** [comment]
@briandeconinck & @sh0wn Can we consider this one complete and close it?

**Comment by amylai-va** [comment]
@bacitracin **Non-urgent:** This feedback is for VA Letters and Documents. The previous team decided not to change the link text to reflect the choices made in the checkbox group, but I don't recall why that decision was made. Is this something you'd recommend revisiting and creating a ticket in our backlog for? cc: @Adrian-Brewster

**Comment by amylai-va** [comment]
@sh0wn thanks for the info! (also hello!)
